data-prepper-plugins/opensearch/src/test/java/org/opensearch/dataprepper/plugins/sink/opensearch/BulkRetryStrategyTests.java

```diff
@@ -68,6 +69,10 @@ public class BulkRetryStrategyTests {
 
     @BeforeEach
     public void setUp() {
+        sink = mock(OpenSearchSink.class);
```

Comment:
Using some of the other suggestions, all of these changes should be unnecessary.